IntegrationTests/Services/AWSS3IntegrationTests/ProcessAWSCredentialIdentityResolverTests.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 import XCTest
```

Comment:
New import statements were added following new organization. No additional comments will be written for this type of change.